Common/Securities/SymbolProperties.cs

```diff
@@ -72,10 +72,18 @@ public string MarketTicker
             get;
         }
 
+        /// <summary>
+        /// The minimum order size allowed
+        /// </summary>
```

Comment:
For pairs like forex or crypto we should specify in the docs in what unit is this expected to be expressed in, believe this is in the Base currency, right?